### PR TITLE
feat(service): Add Viber

### PR DESCRIPTION
**Pull request by irfansofyana**
## Description

Implement Viber service to solve #114 

## Motivation and Context
- This PR is to solve #114 

## How Has This Been Tested?
- Unit Test
image

- End-to-end testing
image

## Screenshots / Output (if appropriate):
- Screenshot End to end testing
The script that is used can be seen in the README with some modifications to change dummy values.
image

- Unit Test
image

- All Test Results
There is a failure in `syslog` service but it's not related with this PR, maybe we can solve it in different PR?
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (no code change)
- [ ] Refactor (refactoring production code)
- [ ] Other 

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: service/viber/README [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# viber
```
Comment (svaloumas) [inline service/viber/README:1]:
```suggestion
# Viber 

[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white&style=flat)](https://pkg.go.dev/github.com/nikoksr/notify/service/viber)
```

File: service/viber/viber_test.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
+package viber
```
[the remark this answers is not in the document]
Comment (irfansofyana) [reply service/viber/viber_test.go:1]: Hi @svaloumas sure! thank you for your review. I will go through your review.

**Comment by irfansofyana** [comment]
hey @svaloumas thanks for your time on reviewing this. I've addressed your comments, please help to re-review it again, thank you! 

also @nikoksr would appreciate if you have any comment on this PR. Thank you!

**Comment by nikoksr** [comment]
@irfansofyana I appreciate your hustle. I firmly believe @svaloumas reviews are of a consistent high quality. There won't be additional value my review could provide.

**Comment by svaloumas** [comment]
@irfansofyana Apologies for getting back late to this! Let's just remove this sneaky line [here](https://github.com/nikoksr/notify/pull/415/files#diff-b8b704307ee6122e74f9edc773c4a733f7bad939f57ddee92da45963a839a077R38) that causes the linter failure and we're ready to go! 👍

**Comment by irfansofyana** [comment]
@svaloumas upss sorry for that, it should be fixed now. Thanks!

**Review by svaloumas — APPROVED** [review]

**Pull request merged**